web-components/src/components/chip/Chip.ts

```diff
@@ -145,6 +145,18 @@ export namespace Chip {
     };
 
     handleClick(e: MouseEvent) {
+      const tooltip = this.shadowRoot?.querySelector("md-tooltip");
+      tooltip?.dispatchEvent(
+        new CustomEvent("tooltip-destroy", {
+          bubbles: true,
+          composed: true,
+          detail: {
+            placement: tooltip?.placement,
+            reference: tooltip?.reference,
+            popper: tooltip?.popper
+          }
+        })
+      );
```

Comment:
What about on Keyboard enter/space key? As the chip can be removed using keyboard navigation too.

Reply:
Same handleClick method is called with keyboard navigation also